src/explore-education-statistics-admin/src/pages/methodology/edit-methodology/status/MethodologyStatusPage.tsx

```diff
@@ -76,6 +80,15 @@ const MethodologyStatusPage = () => {
 
     onMethodologyChange(nextSummary);
 
+    const [nextCanApprove, nextCanMarkAsDraft] = await Promise.all([
+      permissionService.canApproveMethodology(methodologyId),
+      permissionService.canMarkMethodologyAsDraft(methodologyId),
+    ]);
+
+    setPermissions({
+      value: { canApprove: nextCanApprove, canMarkAsDraft: nextCanMarkAsDraft },
+    });
```

Comment:
`useAsyncRetry` exposes a `retry` method that can be used instead of all this (you might want to alias it as `refreshPermission` or something)

Reply:
That's great to know thanks as I was looking through existing code for ideas on how to avoid that duplication! Just made that change. Hopefully I have interpreted this comment correctly 🙂